src/core/src/op/irdft.cpp

```diff
@@ -0,0 +1,82 @@
+//*****************************************************************************
+// Copyright 2017-2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/irdft.hpp"
+
+#include <algorithm>
+#include <irdft_shape_inference.hpp>
```

Comment:
Please use "" for our headers
```suggestion
#include "irdft_shape_inference.hpp"
```

Reply:
Fixed.